compiler-plugin/src/test/scala/cc/Test12.scala

```diff
@@ -1,27 +1,52 @@
 package cc
 
+import cps.plugin.*
+
 import org.junit.Test
 
 class Test12 {
 
-  /*
+
   @Test
   def testCompileAndRunM1(): Unit =
     val dotcInvocations = new DotcInvocations()
-    DotcInvocations.compileAndRunFilesInDirAndCheckResult(
-      "testdata/set12/m1",
-      "cpstest.Test12m1",
-      "myurltransformed\n"
-    )
-    val reporter        = dotcInvocations.reporter
-    println("summary: " + reporter.summary)
-   */
+    try {
+      val (code, output) =
+        dotcInvocations.compileAndRunFilesInDir("testdata/set12/m1", "cpstest.Test12m1")
+      val reporter       = dotcInvocations.reporter
+      println("summary: " + reporter.summary)
+      println(s"output=${output}")
+      assert(false, "Expected exception was not thrown")
+    } catch {
+      case ex: CpsTransformException =>
```

Comment:
Actually we don't want to throw exception from invocation.
(it's just forgotten try/catch)
Thr correct way is to catch CpsTransformException and use report.error for diagnostic.

Reply:
But the `DotcInvocations.compileAndRunFilesInDirs` method throws RuntimeException when compilation is failed

Reply:
Ugu...  let's think, 
  Mb create subclass of Runtime exception for compilation error and check one ?